python/lsst/obs/base/gen3/calibRepoConverter.py

```diff
@@ -81,7 +81,7 @@ def addDatasetType(self, datasetTypeName, storageClass):
                 "instrument": self.baseDataId["instrument"],
                 "calibration_label": label,
                 "valid_first": datetime.strptime(row["validStart"], "%Y-%m-%d"),
-                "valid_last": datetime.strptime(row["validEnd"], "%Y-%m-%d"),
+                "valid_last": datetime.strptime(row["validEnd"], "%Y-%m-%d") + timedelta(day=1),
```

Comment:
@SimonKrughoff may correct me but defects are imminently going to have validity ranges of YYYY-MM-DDTHH:MM:SS.

Reply:
All time stamps (flats, darks, defects, ...) need to have sub-day resolution;  seconds is overkill but seems fine.

Reply:
I'd like to always ingest defects directly into Gen3 repo rather than going through a Gen2 repo, so this code path won't apply to them.